internal/cmd/agent.go

```diff
@@ -48,10 +50,20 @@ coder agent start
 coder agent start --coder-url https://my-coder.com --token xxxx-xxxx
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var (
-				ctx = cmd.Context()
-				log = slog.Make(sloghuman.Sink(os.Stderr)).Leveled(slog.LevelDebug)
-			)
+			ctx := cmd.Context()
+			sinks := []slog.Sink{
+				sloghuman.Sink(os.Stderr),
+			}
+
+			file, err := os.OpenFile(filepath.Join(os.TempDir(), "coder-agent.log"), os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
+			if err == nil && file != nil {
+				sinks = append(sinks, slogjson.Sink(file))
```

Comment:
slogjson is very hard to read, and we don't have automated ingestion of this log file into stackdriver so I'd rather stick with sloghuman for now. We can change this later with no consequences

Reply:
We don't have tools for consuming slogjson output and converting to sloghuman output? The slogjson output seems to have more info (e.g. full paths to files) and being machine-readable does seem potentially useful.

I don't have strong feelings about it though, I'll change to sloghuman :man_shrugging:

Reply:
Yeah that's the only difference between them, they have the same details otherwise. coder-cli doesn't have many files so I'm not too worried about not being able to find the files due to conflicts